06_puppy/n0npax/store_test.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storerImpl int
+
+const (
+	syncStorer storerImpl = iota
+	memStorer  storerImpl = iota
```

Comment:
You don't need to specify this as two `iota`'s. You can just define this as below and they will be seperate enum values.

```go
const (
    syncStorer storerImpl = iota
    memStorer
)
```

Reply:
fixed. Thanks